cpp/msbuild/ice.cpp.props

```diff
@@ -81,7 +81,7 @@
       <!-- PDB settings -->
       <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
       <ProgramDataBaseFileName>$(IntDir)$(TargetName)-objs.pdb</ProgramDataBaseFileName>
-      <LanguageStandard>stdcpp17</LanguageStandard>
+      <LanguageStandard>stdcpp20</LanguageStandard>
```

Comment:
This file is included by all the projects, I think we can remove the rest of the `LanguageStandard` settings.

At some point we build compilers/python with a different setting and that is why we also have it in other projects.

Reply:
Removed these redundant settings.